### PR TITLE
[management] pretty print operational tool

### PR DESCRIPTION
**Pull request by gregnazario**
I mistakingly pushed these to the wrong branch, so this should update the info from those two PRs to master
https://github.com/libra/libra/pull/5399
https://github.com/libra/libra/pull/5392

### CODE REVIEW

File: config/management/operational/src/validator_set.rs [hunk 1]
```diff
@@ -18,9 +24,56 @@ pub struct ValidatorSet {
...
+                })
+                .collect()
+        })
+    }
+}
+
+#[derive(Serialize)]
+pub struct DecryptedValidatorInfo {
+    pub account_address: AccountAddress,
+    // Validator config
```
Comment (davidiw) [inline config/management/operational/src/validator_set.rs:48]: might be more effective if you had ValidatorConfig but a single noun like this isn't particularly clear what it implies... from my high context perspective I take it to mean the following fields are derived from the validator config;

File: config/management/operational/src/command.rs [hunk 2]
```diff
@@ -108,23 +109,57 @@ impl std::fmt::Display for CommandName {
...
+            Command::ExtractPrivateKey(cmd) => Self::print_success(cmd.execute()),
+            Command::ExtractPublicKey(cmd) => Self::print_success(cmd.execute()),
+            Command::PrintAccount(cmd) => Self::pretty_print(cmd.execute()),
+            Command::RemoveValidator(cmd) => Self::pretty_print(cmd.execute()),
+            Command::RotateConsensusKey(cmd) => Self::print_transaction_context(cmd.execute()),
+            Command::RotateOperatorKey(cmd) => Self::print_transaction_context(cmd.execute()),
+            Command::RotateFullNodeNetworkKey(cmd) => {
+                Self::print_transaction_context(cmd.execute())
+            }
+
```
Comment (davidiw) [inline config/management/operational/src/command.rs:123]: weird whitespace in the sea of none

File: config/management/operational/src/validator_set.rs (hunk 1 above)
Comment (gregnazario) [reply config/management/operational/src/validator_set.rs:48]: Yeah, they're derived from ValidatorConfig, and one point it was the validator config, but it's been cut out.

File: config/management/operational/src/command.rs (hunk 2 above)
Comment (gregnazario) [reply config/management/operational/src/command.rs:123]: strange

**Pull request closed without merging**